Piwik.Tracker/PiwikTracker.cs

```diff
@@ -1280,6 +1279,7 @@ protected bool LoadVisitorIdCookie()
             {
                 _visitCount = long.Parse(parts[2]);
             }
+            // todo _currentVisitTs is never used (was earlier assigned with null in constructor)

```

Comment:
It is not being used by anyone at the moment so there is maybe no need to add the getter.

I am saying this because if we follow this path then we might need to add getters for many more attributes.

Is it bad practice to leave the field unused? I was thinking of keeping it in the code to document the fact it exists.

Maybe we can replace it by a comment describing the content of the cookie parts?